### PR TITLE
Task result sam

### PR DESCRIPTION
**Pull request by samiur98**
- Task Review Page Implemented
- Quiz Review Component Implemented

### CODE REVIEW

File: src/Components/TaskReview/QuizReview.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.quiz-review {
```
Comment (Bart-Ilg) [inline src/Components/TaskReview/QuizReview.css:1]: good

File: src/Components/TaskReview/TaskReview.js [hunk 2]
```diff
@@ -24,6 +44,38 @@ export default function TaskReview(props) {
       </div>))
   }
 
+  function renderOptions() {
```
Comment (Bart-Ilg) [inline src/Components/TaskReview/TaskReview.js:47]: format was generally  easy to work with, worked with it in my branch for combined effort

File: src/Components/TaskReview/TaskReview.js [hunk 3]
```diff
@@ -32,29 +84,23 @@ export default function TaskReview(props) {
...
-          <QuestionReview questionAndAnswer={focusedQA}/>
-        </div>
-        <div className="column">
-          <h1>Requirements</h1>
-          <h2>{mapRequirements(task.requirements)}</h2>
-        </div>
+
+  function getButtons() {
+    return(
+      <div className='task-review-buttons'>
```
Comment (Bart-Ilg) [inline src/Components/TaskReview/TaskReview.js:90]: good and condensed

File: src/Components/TaskReview/TaskReviewStyles.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
+export const oBarGrayStyle = {
```
Comment (Bart-Ilg) [inline src/Components/TaskReview/TaskReviewStyles.js:1]: neat

**Review by Bart-Ilg — APPROVED** [review]
Overall good code. I worked with alot of this on my branch since we're working on the same page and its been easy to work with

File: src/Components/TaskReview/QuizReview.js [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+        if((option === selected) && (selected !== correctOption)) {
+            return optionRedStyle;
+        }
+        if(option === selected) {
+            return optionGreenStyle;
+        }
+        if(option === correctOption) {
+            return optionGreenStyle;
+        }
+        return optionGrayStyle;
```
Comment (ConnorBoulais) [inline src/Components/TaskReview/QuizReview.js:82]: Good job rendering the different question colors for correct/incorrect. Looks really good

**Review by ConnorBoulais — APPROVED** [review]
Great job on the task review, it looks just like the design figma!

**Pull request merged**